habitat-baselines/habitat_baselines/rl/ppo/ppo_trainer.py

```diff
@@ -162,27 +162,27 @@ def _setup_actor_critic_agent(self, ppo_cfg: "DictConfig") -> None:
             self.config.habitat_baselines.rl.ddppo.pretrained_encoder
             or self.config.habitat_baselines.rl.ddppo.pretrained
         ):
-            pretrained_state = torch.load(
+            pretrained_state = self.load_checkpoint(
                 self.config.habitat_baselines.rl.ddppo.pretrained_weights,
                 map_location="cpu",
             )
-
-        if self.config.habitat_baselines.rl.ddppo.pretrained:
-            self.actor_critic.load_state_dict(
-                {  # type: ignore
-                    k[len("actor_critic.") :]: v
-                    for k, v in pretrained_state["state_dict"].items()
-                }
-            )
-        elif self.config.habitat_baselines.rl.ddppo.pretrained_encoder:
```

Comment:
config.habitat_baselines.rl.ddppo.pretrained_encoder is no longer used ?

Reply:
We check `if (self.config.habitat_baselines.rl.ddppo.pretrained_encoder or self.config.habitat_baselines.rl.ddppo.pretrained): ` in line 161.  

So at the point you commented either one of keys will be True or both of them will be True.

Since the `self.config.habitat_baselines.rl.ddppo.pretrained` has "higher priority" we chec it first even if both keys are set to True, else we load the `pretrained_encoder` keys so the logic doesn't change.